migrate.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# Use this tool to assist in migrating your Orb Development Kit pipeline.
+
+verify_run() {
+  # Ensure .circleci/config.yml exists
+  if [ ! -f .circleci/config.yml ]; then
+    echo "No .circleci/config.yml found"
+    echo "This does not appear to be the root of a CircleCI project"
+    exit 1
+  fi
+}
+
+backup_contents() {
+  # Backup the existing files in .circleci/
+  for file in .circleci/*; do
+    if [ -f "$file" ]; then
+      mv "$file" "$file.bak"
+    fi
+  done
+}
+
+download_template() {
+  ORB_TEMPLATE_VERSION=$(curl -Ls -o /dev/null -w "%{url_effective}\n" "https://github.com/CircleCI-Public/Orb-Project-Template/releases/latest" | sed 's:.*/::' | xargs)
+  ORB_TEMPLATE_DOWNLOAD_URL="https://github.com/CircleCI-Public/Orb-Project-Template/archive/refs/tags/${ORB_TEMPLATE_VERSION}.tar.gz"
+  ORB_TEMP_DIR=$(mktemp -d)
+
+  curl -Ls "$ORB_TEMPLATE_DOWNLOAD_URL" -o "$ORB_TEMP_DIR/orb-project-template.tar.gz"
+  tar -xzf "$ORB_TEMP_DIR/orb-project-template.tar.gz" -C "$ORB_TEMP_DIR"
+  cp -r "${ORB_TEMP_DIR}/orb-project-template/.circleci/*" .circleci/
+}
+
+copy_custom_components() {
+  ORIGINAL_EXECUTORS=$(yq '.executors' .circleci/config.yml.bak)
+  ORIGINAL_JOBS=$(yq '.jobs' .circleci/config.yml.bak)
+  ORIGINAL_COMMANDS=$(yq '.commands' .circleci/config.yml.bak)
+  export ORIGINAL_EXECUTORS
```

Comment:
Any reason why we have to export these? I am not seeing these variables being used anywhere else.

Reply:
Good question, I vaguely remember when I added this and I think it was before maybe rewriting some things. I don't believe this has any purpose. Removing. Thank you.